tasks.py

```diff
@@ -611,11 +589,28 @@ def pylint(context, target=None, recursive=False):
         run_command(context, command)
 
 
-@task
-def ruff(context, output_format="text"):
-    """Run ruff to perform static analysis and linting."""
-    command = f"ruff --output-format {output_format} development/ examples/ nautobot/ tasks.py"
-    run_command(context, command)
+@task(
+    help={
+        "action": "One of 'lint', 'format', or 'both'",
+        "autoformat": "Automatically apply formatting recommendations, rather than failing if formatting is incorrect.",
+        "fix": "Automatically apply linting recommendations where possible. May not be able to fix all.",
+        "output_format": "see https://docs.astral.sh/ruff/settings/#output-format",
+    },
+)
+def ruff(context, action="both", autoformat=False, fix=False, output_format="text"):
```

Comment:
When I run `invoke ruff` it's only running the format command.

Reply:
Strange, it's working here:

```
❯ invoke ruff
Running docker compose command "ps --services --filter status=running"
Running docker compose command "run --rm --entrypoint 'ruff format --check development/ examples/ nautobot/ tasks.py' 
nautobot"
PYTHON_VER=3.8 \
docker compose \
    --project-name "nautobot" \
    --project-directory "/Users/glennmatthews/Documents/git-stuff/nautobot/development/" \
    -f "/Users/glennmatthews/Documents/git-stuff/nautobot/development/docker-compose.yml" \
    -f "/Users/glennmatthews/Documents/git-stuff/nautobot/development/docker-compose.postgres.yml" \
    -f "/Users/glennmatthews/Documents/git-stuff/nautobot/development/docker-compose.dev.yml" run \
    --rm \
    --entrypoint 'ruff format \
    --check development/ examples/ nautobot/ tasks.py' nautobot
[+] Running 3/0
 ⠿ Container nautobot-selenium-1  Running                                                                          0.0s
 ⠿ Container nautobot-db-1        Running                                                                          0.0s
 ⠿ Container nautobot-redis-1     Running                                                                          0.0s
838 files already formatted
Running docker compose command "ps --services --filter status=running"
Running docker compose command "run --rm --entrypoint 'ruff check --output-format text development/ examples/ nautobot/ 
tasks.py' nautobot"
PYTHON_VER=3.8 \
docker compose \
    --project-name "nautobot" \
    --project-directory "/Users/glennmatthews/Documents/git-stuff/nautobot/development/" \
    -f "/Users/glennmatthews/Documents/git-stuff/nautobot/development/docker-compose.yml" \
    -f "/Users/glennmatthews/Documents/git-stuff/nautobot/development/docker-compose.postgres.yml" \
    -f "/Users/glennmatthews/Documents/git-stuff/nautobot/development/docker-compose.dev.yml" run \
    --rm \
    --entrypoint 'ruff check \
    --output-format text development/ examples/ nautobot/ tasks.py' nautobot
[+] Running 3/0
 ⠿ Container nautobot-redis-1     Running                                                                          0.0s
 ⠿ Container nautobot-db-1        Running                                                                          0.0s
 ⠿ Container nautobot-selenium-1  Running                                                                          0.0s
```

Reply:
In your case is it perhaps failing the `ruff format` and aborting early?

Reply:
That's exactly what it was. Thanks!